.travis.yml

```diff
@@ -25,6 +25,7 @@ before_deploy:
 - >
   if ! [ "$BEFORE_DEPLOY_RUN" ]; then
     export BEFORE_DEPLOY_RUN=1;
+    export ASCIIDOCTOR_CORE_VERSION=v1.5.6;
```

Comment:
That's a good idea.
Though, it will not be overridden by the `env` defined in `.travis.yml` ?